eclipse-collections/src/main/java/org/eclipse/collections/impl/bag/sorted/mutable/SynchronizedSortedBag.java

```diff
@@ -716,4 +717,12 @@ public int detectLastIndex(Predicate<? super T> predicate)
             return this.getDelegate().getLastOptional();
         }
     }
+
+    @Override
+    public MutableSortedSet<T> selectUnique()
+    {
```

Comment:
Same comment as primitive side. Please wrap this around a synchronized block and delegate it.